RecoHI/HiJetAlgos/plugins/HiFJRhoFlowModulationProducer.cc

```diff
@@ -90,128 +172,202 @@ void HiFJRhoFlowModulationProducer::produce(edm::Event& iEvent, const edm::Event
     });
   }
 
+  // If jetty regions are excluded from the flow fits, read the jet collection
   edm::Handle<reco::JetView> jets;
-  if (doJettyExclusion_)
+  if (doJettyExclusion_) {
     iEvent.getByToken(jetToken_, jets);
+  }
 
   int nFill = 0;
 
-  double eventPlane2 = -100;
-  double eventPlane3 = -100;
+  // Initialize arrays for event planes
+  const int nFlow = lastFittedVn_ - firstFittedVn_ +
+                    1;  // Number of flow components in the fit to particle flow condidate distribution
+  double eventPlane[nFlow];
+  for (int iFlow = 0; iFlow < nFlow; iFlow++)
+    eventPlane[iFlow] = -100;
 
-  constexpr int nParamVals = 9;
+  // Initialize the output vector with flow fit components
+  const int nParamVals =
```

Comment:
Why spread this over several lines?

Reply:
I had it in one line, but 'scram build code-format' made this change. I do not know why, since it looks much better in one line. Maybe the code formatter only allows certain number of operations in one line?

Reply:
With the comment at the end the code-format find that the whole line is too much long. If you move the comment onto a different line the rest will remain in one single line